.drone/drone.yml

```diff
@@ -1,6 +1,8 @@
 ---
 kind: pipeline
 name: loki-build-image
+node:
```

Comment:
not exactly sure why this is needed here

Reply:
CI failed three times; it was an experiment suggested by Julien.